lib/components/UserActionsUI.jsx

```diff
@@ -2,24 +2,33 @@ import React from 'react'
 
 import { FormPanel } from 'lib/components/FormPanel'
 import { DepositUI } from 'lib/components/DepositUI'
+import { PermitAndDepositUI } from 'lib/components/PermitAndDepositUI'
 import { SweepTimelockedUI } from 'lib/components/SweepTimelockedUI'
 import { UnlockDepositUI } from 'lib/components/UnlockDepositUI'
 import { WithdrawUI } from 'lib/components/WithdrawUI'
 
 export const UserActionsUI = (props) => {
+  let depositUI
+  if (props.poolAddresses && props.poolAddresses.token.toLowerCase() === '0x6b175474e89094c44da98b954eedeac495271d0f') {
```

Comment:
What is the hard-coded address here? local buidler mock?

Also, this can be re-written to `if (props?.poolAddresses?.token?.toLowerCase() === '0xface...')`

Reply:
That's the Dai mainnet address.

I'll put it out so that it has a name, and is more intention-revealing